backend/campaigns/admin.py

```
@@ -0,0 +1,169 @@
+            '<table>{}</table>'
+            '<p style="color:#666;">Source: Portal DB. Hits are redirect requests, not unique '
+            'visitors; use GA for session-level traffic.</p>',
+            body,
+        )
+
+    def save_model(self, request, obj, form, change):
+        if not change and not obj.created_by:
+            obj.created_by = request.user
+        super().save_model(request, obj, form, change)
+
+    def save_formset(self, request, form, formset, change):
```

Ruff (0.16.0)

[warning] Unused method argument: `form`

(ARG002)